configuration.json.dist

```diff
@@ -0,0 +1,21 @@
+{
+    "storage": {
+        "active": "imagekit",
+        "imagekit": {
+            "enableDatedFolders": true,
+            "auth": {
+                "privateKey": "your-private-key",
+                "publicKey": "your-public-key",
+                "urlEndpoint": "https://ik.imagekit.io/your-imagekit-id"
+            },
+            "uploadoptions": {
```

Comment:
`uploadOptions`
Please double check all keys in config.

Reply:
done